src/iptvsimple/utilities/FileUtils.h

```diff
@@ -33,8 +33,7 @@ namespace iptvsimple
     {
     public:
       static std::string PathCombine(const std::string& path, const std::string& fileName);
-      static std::string GetClientFilePath(const std::string& fileName);
-      static std::string GetUserFilePath(const std::string& fileName);
+      static std::string GetUserDataAddonFilePath(const std::string& fileName);
       static int GetFileContents(const std::string& url, std::string& content);
```

Comment:
I have seen at other places that you also use `XBMC->DeleteFile`. Maybe it's a good idea to add a `DeleteFile` method to this class to encapsulate access also to `DeleteFile` and esp. have the ugly "if windows undef DeleteFile" hack only at one place.

Reply:
Done